masterlist.yaml

```
@@ -396,6 +396,12 @@ plugins:
         name: 'Blackout Kodama Skin'
     group: *bgsCreationsGroup
 
+  - name: 'sfbgs00b.esm'
+    url:
+      - link: 'https://creations.bethesda.net/en/starfield/details/1106af41-fac5-431e-aa1a-163b3e5d2892/Escape/'
+        name: 'Escape'
+    group: *bgsCreationsGroup
+    
   - name: 'sfbgs00e.esm'
```

There are some extra spaces here:

```suggestion
    group: *bgsCreationsGroup

  - name: 'sfbgs00e.esm'
```